scripts/binary_release.sh

```diff
@@ -53,6 +53,7 @@ function upload_binary {
 
   else
     folder="${release%-release}"
+    aws s3 cp --acl public-read target/release/$1 s3://build.nearprotocol.com/nearcore/${os}/${BRANCH}/${folder}/$1
```

Comment:
how is this script different from the github action? is it just a way to do the same manually?

Reply:
You could run it manually, but you would need aws credentials, in the GH action, they are loaded here:
```
- name: Configure AWS Credentials
        uses: aws-actions/configure-aws-credentials@v4
        with:
          role-to-assume: arn:aws:iam::754641474505:role/GitHubActionsRunner
          aws-region: us-west-1
```